src/services/photos/utils/index.ts

```diff
@@ -0,0 +1 @@
+export * from './photosUtils';
```

Comment:
if the utils files are inside the `photos/utils` folder, maybe is better to name it 'utils.ts', just for the sake of making the file trees shorter and easier to navigate